### PR TITLE
New Cellrangerarc modules

### PR DESCRIPTION
**Pull request by heylf**
## PR checklist

Closes #XXX 

- [X] This comment contains a description of changes (with reason).
- [] If you've fixed a bug or added code that should be tested, add tests!
- [X] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [X] Remove all TODO statements.
- [X] Emit the `versions.yml` file.
- [X] Follow the naming conventions.
- [X] Follow the parameters requirements.
- [X] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

**Comment by heylf** [comment]
Waiting for docker containers by nf-core.

File: modules/nf-core/cellrangerarc/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Dockerfile to create container with Cell Ranger v2.0.2
+# Push to nfcore/cellranger-arc:<VER>
```
Comment (maxulysse) [inline modules/nf-core/cellrangerarc/Dockerfile:2]:
```suggestion
# Push to quay.io/nf-core/cellranger-arc:<VER>
```

File: modules/nf-core/cellrangerarc/mkfastq/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# Dockerfile to create container with Cell Ranger Arc v2.0.2 and bcl2fastq v2.20.0
+# Push to nfcore/cellranger-arc-mkfastq:<VER>
```
Comment (maxulysse) [inline modules/nf-core/cellrangerarc/mkfastq/Dockerfile:2]:
```suggestion
# Push to quay.io/nf-core/cellranger-arc-mkfastq:<VER>
```

File: modules/nf-core/cellrangerarc/README.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+2. Edit the Dockerfile. Update the Cell Ranger Arc versions in this line:
+
+```bash
+ENV CELLRANGER_ARC_VER=<VERSION>
+```
+
+3. Create and test the container:
+
+```bash
+docker build . -t nfcore/cellranger-arc:<VERSION>
```
Comment (maxulysse) [inline modules/nf-core/cellrangerarc/README.md:16]:
```suggestion
docker build . -t quay.io/nf-core/cellranger-arc:<VERSION>
```

File: modules/nf-core/cellrangerarc/README.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+```
+
+4. Access rights are needed to push the container to the Dockerhub nfcore organization, please ask a core team member to do so.
+
+```bash
+docker push nfcore/cellranger-arc:<VERSION>
```
Comment (maxulysse) [inline modules/nf-core/cellrangerarc/README.md:22]:
```suggestion
docker push quay.io/nf-core/cellranger-arc:<VERSION>
```

File: modules/nf-core/cellrangerarc/mkfastq/README.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+```bash
+ENV BCL2FASTQ2_VER=<VERSION> \
+    ARC_VER=<VERSION>
+```
+
+3. Create and test the container:
+
+```bash
+docker build . -t nfcore/cellranger-arc-mkfastq:<CELLRANGERARC_VERSION>
```
Comment (maxulysse) [inline modules/nf-core/cellrangerarc/mkfastq/README.md:17]:
```suggestion
docker build . -t quay.io/nf-core/cellranger-arc-mkfastq:<CELLRANGERARC_VERSION>
```

**Comment by maxulysse** [comment]
Can you please check the conflicts?

**Review by maxulysse — APPROVED** [review]

**Comment by heylf** [comment]
@grst 

In the scrnaseq I just added to the count/main.nf and mkref/main.nf calls fr python scripts to generate the lib.csv (for count) and config (for mkref). This makes sense for the pipeline but from a pure module perspective this is I think not relevant. What do you think?

Else everything has not changes for the modules.

**Comment by heylf** [comment]
@grst fixed. I put now the python script into the module itself. Modules are now in sync with modules of scrnaseq.

**Review by grst — APPROVED** [review]
LGTM! Thanks @heylf!

**Pull request merged**